### DISCUSSION TITLE
Is Play Activity no longer tracking for everyone? latest beta

### DISCUSSION THREAD

**Discussion by KickPuncher9898** [OnionUI/Onion#878] [Q&A]
Since I believe beta 3 my play activity hasn't been updating. Playing a new game for hours will result in 0m played on the Game Switcher overlay, and my Activity Tracker app is bugged with a large negative total playtime and inaccurate entries. I see it mentioned by others on the reddit so I assumed it was a known issue, but here I don't see an open or closed Issue for it. 

Yesterday I tried using a PlayActivity backup, same result. Tried a PlayActivity file from a beta 2 install and still same thing.

**Comment by Aemiii91** [comment]
Yep, it's "out of service" at the moment. I'm working on fixing it 👌

**Reply by KickPuncher9898** [reply]
Ah good to hear! Thank you!

**Reply by KickPuncher9898** [reply]
Am I able to downgrade to beta 2 by dropping the install files in? Would that revert me and restore activity tracking?

**Reply by Aemiii91** [reply]
It should work, yes

**Reply by KickPuncher9898** [reply]
Hey it worked! And my games with too many characters and/or " ' " in the title are back! Thanks for your help.
